doc/src/fix_reaxff_species.rst

```diff
@@ -110,10 +110,10 @@ all types from 1 to :math:`N`.  A leading asterisk means all types from
 The optional keyword *element* can be used to specify the chemical
 symbol printed for each LAMMPS atom type. The number of symbols must
 match the number of LAMMPS atom types and each symbol must consist of
-1 or 2 alphanumeric characters. Normally, these symbols should be
-chosen to match the chemical identity of each LAMMPS atom type, as
-specified using the :doc:`reaxff pair_coeff <pair_reaxff>` command and
-the ReaxFF force field file.
+1 or 2 alphanumeric characters. By default, these symbols are the same
+as the chemical identity of each LAMMPS atom type, as specified by the
+:doc:`reaxff pair_coeff <pair_reaxff>` command and the ReaxFF force
```

Comment:
ReaxFF, since it is the name of the FF, not the name of the LAMMPS command.

Reply:
I am not sure what you want changed here, if anything

Reply:
oh, I see your point. will correct